### PR TITLE
gawb-3025 permanently hide free credits termination banner

### PR DESCRIPTION
**Pull request by zarsky-broad**
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [x] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [x] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [x] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [x] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [x] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [x] **LR** sign off
- [x] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Review by MattPutnam — APPROVED** [review]
👍 , didn't run this myself but code looks good.

File: automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/user/FreeTrialSpec.scala [hunk 1]
```diff
@@ -127,10 +127,10 @@ class FreeTrialSpec extends FreeSpec with BeforeAndAfterEach with Matchers with
       }
     }
 
-    "Disabled" - {
+    "Finalized" - {
       "should not see the free trial banner" in withWebDriver { implicit driver =>
         registerCleanUpForDeleteTrialState()
-        Thurloe.keyValuePairs.set(subjectId, "trialState", "Disabled")
+        Thurloe.keyValuePairs.set(subjectId, "trialState", "Finalized")
```
Comment (davidangb) [inline automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/user/FreeTrialSpec.scala]: we need both Disabled and Finalized tests!

**Review by davidangb — COMMENTED** [review]
works for me locally, one comment about tests, nothing else to say on the code side.

On the UX side: When I saw the "X" with "Hide forever" beneath it, I assumed that the words "Hide forever" were explaining what the "X" icon would do; I didn't think the two links did different things.

**Review by andy7i — APPROVED** [review]
👍 for tests

**Comment by zarsky-broad** [comment]
@MattPutnam @davidangb I've brought over my `add-hover-style` utility from [zarsky-broad/dmohs-react-heroes](https://github.com/zarsky-broad/dmohs-react-heroes), take a look and make sure it looks ok to you, please and thank you.

**Comment by zarsky-broad** [comment]
jenkins retest

**Comment by MattPutnam** [comment]
I'm not wild about the CSS hack, but 👍

**Review by davidangb — APPROVED** [review]
👍 pending tests!

**Pull request merged**